README.md

```diff
@@ -17,6 +17,15 @@ make -j`nproc`
 make install
 ```
 
+## Install (Mac)
+Add the following to the very top of the file `canokey-core/canokey-crypto/mbedtls/library/CMakeLists.txt`:
+```
+if(${APPLE})
+	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-unused-but-set-variable -Wno-unused-but-set-parameter")
+endif()
+```
+When building QEMU with the `--enable-canokey` option add the additional option to point to the static library: `--extra-ldflags='-L/usr/local/lib -lcanokey-qemu'`.  Note: if your libraries from `make install` for canokey-qemu went into a directory other than `/usr/local/lib` update the `--extra-ldflags` path as needed.
```

Comment:
```
--extra-ldflags='-L/usr/local/lib -lcanokey-qemu'
```

Since pkg-config is used, I think these flags should be auto-populated by meson in qemu instead of adding them manually?

Reply:
I just tested and you're correct, with the pkg-config file from this PR the extra-ldflags option isn't necessary.

Reply:
README updated to remove unnecessary steps and to add additional info.